src/devices/cpu/drcbearm64.h

```diff
@@ -0,0 +1,286 @@
+// license:BSD-3-Clause
+// copyright-holders:windyfairy
+#ifndef MAME_CPU_DRCBEARM64_H
+#define MAME_CPU_DRCBEARM64_H
+
+#pragma once
+
+#include "drcuml.h"
+#include "drcbeut.h"
+
+#include "asmjit/src/asmjit/asmjit.h"
+#include "asmjit/src/asmjit/a64.h"
+
+#include <vector>
+
+
+namespace drc {
+
+class drcbe_arm64 : public drcbe_interface
+{
+	typedef uint32_t (*arm64_entry_point_func)(void *entry);
```

Comment:
These are usually more readable as:
```c++
	using arm64_entry_point_func = uint32_t (*)(void *entry);
```

That way the name isn’t stuck in the middle of the type.

Reply:
fwiw this was all cruft inherited from using drcbex64 as the base for my code originally, so all of these points apply to the x86 and x64 backends as well (besides me deleting private for that one constructor).

The changes are simple so I updated all of the headers at the same time: https://github.com/mamedev/mame/pull/13162/commits/5fc82fb26a868fd7851def717e3c13d1e5acfcf3